tests/bluetooth/audio/cap_commander/src/test_broadcast_reception.c

```diff
@@ -32,21 +32,43 @@ LOG_MODULE_REGISTER(bt_broadcast_reception_test, CONFIG_BT_CAP_COMMANDER_LOG_LEV
 struct cap_commander_test_broadcast_reception_fixture {
 	struct bt_conn conns[CONFIG_BT_MAX_CONN];
 
-	struct bt_bap_bass_subgroup start_subgroups[CONFIG_BT_BAP_BASS_MAX_SUBGROUPS];
+	struct bt_bap_bass_subgroup subgroups[CONFIG_BT_BAP_BASS_MAX_SUBGROUPS];
 	struct bt_cap_commander_broadcast_reception_start_member_param
 		start_member_params[CONFIG_BT_MAX_CONN];
 	struct bt_cap_commander_broadcast_reception_start_param start_param;
+	struct bt_cap_commander_broadcast_reception_stop_member_param
+		stop_member_params[CONFIG_BT_MAX_CONN];
+	struct bt_cap_commander_broadcast_reception_stop_param stop_param;
 };
 
+/* src_id can not be part of the fixture since it is accessed in the callback function */
+static uint8_t src_id[CONFIG_BT_MAX_CONN];
+static struct bt_bap_broadcast_assistant_cb broadcast_assistant_cb;
```

Comment:
Ideally we wouldn't have global variables, but I never found a way to access the current test fixture for async functionality like this. 

We do, however, need to clear `src_id` between each test. 

`broadcast_assistant_cb` could/should be moved to the fixture